csharp/test/Drivers/Interop/FlightSql/ClientTests.cs

```diff
@@ -159,12 +159,12 @@ public void CanClientExecuteQueryWithNoResults()
         {
             foreach (FlightSqlTestEnvironment environment in _environments)
             {
-                environment.Query = "SELECT * WHERE 0=1";
+                environment.Query = "SELECT 1 WHERE false"; // The query simulates no results
```

Comment:
`SELECT *` w/o `FROM` clause is invalid syntax in most query dialects, PR replaces this with `SELECT 1` that is more robust